x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -805,6 +805,9 @@ private void assertRollupIndexSettings(String sourceIndex, String rollupIndex, G
             indexSettingsResp.getSetting(rollupIndex, IndexMetadata.SETTING_NUMBER_OF_REPLICAS)
         );
         assertEquals("true", indexSettingsResp.getSetting(rollupIndex, "index.blocks.write"));
+
+        boolean isSourceHidden = indexSettingsResp.getSetting(sourceIndex, IndexMetadata.SETTING_INDEX_HIDDEN) == "true";
```

Comment:
Maybe

```
if ("true".equals(indexSettingsResp.getSetting(sourceIndex, IndexMetadata.SETTING_INDEX_HIDDEN))) {
...
}
```

Reply:
Fixed in 625f86acce7de12945efeaa359ed0696554881bf